src/features/toilet/ToiletCard.js

```diff
@@ -68,7 +68,7 @@ const StyledToiletCard = styled.div`
 function ToiletCard({ toilet, distance, time }) {
   const navigate = useNavigate();
   const {
-    toiletID,
+    _id,
```

Comment:
화장실 정보를 그대로 받아서 활용할 경우 이렇게 활용하게될거 같습니다.
더불어 _ 린트 경고 이슈도 함께 맞물려 있습니다.

Reply:
_id만 있을때는 정확히 어떤 id 인지 알기 어려울 수 도 있을거 같아요. destructuring 할때 아래와 같이 변수명을 정해서 toiletId 로 받아오는건 어떨까요?

```suggestion
    _id: toiletId,
```

Reply:
아 그리고 위에서 제가 제안드린대로 해도 린트가 오류가 날수도 있을거같은데 그렇다면 확실히 언더바로 변수선언을 막는 룰은 아예 저희 프로젝트에서 제외시키는게 나을수도 있을거 같아요.

Reply:
위 방법도 좋네요~ ㅎㅎ

Reply:
수정완료!